.gitignore

```diff
@@ -29,3 +29,8 @@ src/shared/telemetry/clienttelemetry.d.ts
 # Generated by tests
 src/testFixtures/**/bin
 src/testFixtures/**/obj
+
+# Icons
+resources/icons/cloud9/generated/**
+resources/fonts/aws-toolkit-icons.woff
+media/css/icons.css
```

Comment:
is there a reason we have both `resources/` and `media/` ? is this PR a good time to merge them (e.g. delete `media/`)?

Reply:
Yup agreed.